### PR TITLE
feat: new page overview

### LINKED ISSUE #2: Create new page: Overview
- ignore search bar
- add handler to create a favorite by id

- list
  - popular: places order by popularity
  - ~featured~: ignore 
  - my favorite: only for logged in users, places filter by my favorites, order by last update
  - recently update: places order by last update
  - ~friend activity~: ignore
  - pois: places filter by position, order by last update

- see all target
  - popular => places all, order by most popular
  - ~featured~ => ignore 
  - my favorite  => my places
  - recently update => places all, order by last update
  - pois: places poi, order by last update
  
Screen Shot 2022-09-01 at 19 55 00

### PR DESCRIPTION
**Pull request by braianj**

### CODE REVIEW

File: src/hooks/Place.ts [hunk 1]
```diff
@@ -16,3 +18,39 @@ export function usePlaceId(placeId?: string | null) {
...
+  offset: number
+}) {
+  const [account] = useAuthContext()
+  return useAsyncMemo(async () => {
+    return account && Places.get().getPlacesMyFavorites(options)
+  }, [options?.limit, options?.offset, account])
+}
+
+export function usePlaceListPois(options?: { limit: number; offset: number }) {
+  const [pois] = useAsyncMemo(getPois)
```
Comment (2fd) [inline src/hooks/Place.ts:51]:
```suggestion
```

File: src/hooks/Place.ts [hunk 2]
```diff
@@ -16,3 +18,39 @@ export function usePlaceId(placeId?: string | null) {
...
+  return useAsyncMemo(async () => {
+    console.log({ pois })
```
Comment (2fd) [inline src/hooks/Place.ts:53]:
```suggestion
    const pois = await getPois()
```

**Review by 2fd — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,9 +1,165 @@
...
+      if (account === null) {
+        accountState.select()
+      } else if (place) {
+        const favoritesResponse = await Places.get().updateFavorite(
+          place.id,
+          !place.user_favorite
+        )
+        if (favoritesResponse) {
+          if (placeListLastUpdates && placeListLastUpdates.length > 0) {
+            const placeListLastUpdatesUpdated = placeListLastUpdates?.map(
```
Comment (2fd) [inline src/pages/index.tsx:51]: can we generalize this some way

**Review by 2fd — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 3 above)
Comment (braianj) [reply src/pages/index.tsx]: Please look the new function updatePlaceInPlaceList

**Review by 2fd — APPROVED** [review]

**Pull request merged**